wp-content/themes/access/gulpfile.babel.js

```diff
@@ -98,9 +98,11 @@ gulp.task('sass', () => gulp.src(`${SRC}/scss/style-*.scss`)
     purgecss({
       content: ['./views/**/*.twig'],
       whitelistPatterns: [
+        /fill-[a-z-]*/, // matches "fill-{{ token }}"
```

Comment:
@devowhippit when I remove the `g`(global) it works. 
<img width="1082" alt="Screen Shot 2020-09-10 at 3 52 01 PM" src="https://user-images.githubusercontent.com/14843767/92794585-b1f05280-f37d-11ea-87ba-38b537b78482.png">

when I add the g it doesn't work.